### PR TITLE
Search collection item

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#348

### CODE REVIEW

File: src/common/Grid.tsx [hunk 1]
```diff
@@ -16,6 +17,7 @@ export function Row(props: RowProps) {
         <div
             className={ "Row" + (props.className !== undefined ? " " + props.className : "") }
             style={ props.style }
+            data-testid = { props.dataTestId }
```
Comment (gdethier) [inline src/common/Grid.tsx:20]: The use of test IDs is generally not considered as a good practice (for instance, see [here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text)).

File: src/loc/ContextualizedLocDetails.tsx [hunk 2]
```diff
@@ -287,6 +288,9 @@ export default function ContextualizedLocDetails(props: Props) {
                 locId={ locId }
                 loc={ loc }
             />
+            { loc.locType === 'Collection' &&
+                <CollectionLocItemChecker locId={ locId } />
```
Comment (gdethier) [inline src/loc/ContextualizedLocDetails.tsx:292]: Should be displayed only once collection LOC is closed (see [this comment](https://github.com/logion-network/logion-internal/issues/347#issuecomment-1038892385) from Elie).

File: src/logion-chain/__mocks__/LogionChainMock.ts [hunk 3]
```diff
@@ -23,6 +45,7 @@ const api = {
             asRecovered: () => {},
         }
     },
+    //     const result = await api.query.logionLoc.collectionItemsMap(locId.toHexString(), itemId);
```
Comment (gdethier) [inline src/logion-chain/__mocks__/LogionChainMock.ts:48]: Remove this comment.

**Review by gdethier — APPROVED** [review]
Great job!

File: src/common/Grid.tsx (hunk 1 above)
Comment (benoitdevos) [reply src/common/Grid.tsx:20]: I have removed the usage of test id, but I have never able to convert the test using something else. So I just removed the test too.

File: src/loc/ContextualizedLocDetails.tsx (hunk 2 above)
Comment (benoitdevos) [reply src/loc/ContextualizedLocDetails.tsx:292]: Done.

File: src/logion-chain/__mocks__/LogionChainMock.ts (hunk 3 above)
Comment (benoitdevos) [reply src/logion-chain/__mocks__/LogionChainMock.ts:48]: Done.

**Pull request merged**